ui/src/i18n/pl.json

```diff
@@ -51,27 +51,27 @@
   "accessories.title_accessories": "Akcesoria",
   "accessories.title_accessory_control_disabled": "Sterowanie akcesoriami jest wyłączone",
   "backup.button_download_backup_archive": "Utwórz backup",
-  "backup.button_restore_backup": "Przywróć backup",
+  "backup.button_restore_backup": "Przywróć z backupu",
   "backup.label_archive_name": "Nazwa backupu",
   "backup.label_backup_size": "Rozmiar backupu",
   "backup.label_backup_time": "Czas utworzenia",
   "backup.label_choose_backup_file_to_restore": "Wybierz plik backupu",
   "backup.label_uploading": "Przywracanie…",
   "backup.message_backup_download_failed": "Tworzenie backupu nieudane",
-  "backup.message_backup_exceeds_max_size": "Kopia zapasowa przekracza maksymalny rozmiar pliku przywracania: {{ maxBackupSizeText }}",
+  "backup.message_backup_exceeds_max_size": "Backup przekracza maksymalny rozmiar pliku przywracania: {{ maxBackupSizeText }}",
   "backup.message_backup_help_one": "Utwórz backup całego środowiska Homebridge, który następnie będziesz mógł przywrócic po ponownej instalacji Homebridge, lub po zresetowaniu systemu.",
   "backup.message_backup_help_two": "Wtyczki niedostępne z poziomu repozytorium npm lub instalowane bezpośrednio z GitHub nie będą dodane do backupu.",
   "backup.message_backup_restored": "Przywracanie backupu zakończone sukcesem",
   "backup.message_backup_warning": "Backup zawiera ważne informacje i nie powinien być udostępniany nikomu.",
   "backup.message_restore_failed": "Przywracanie backupu zakończone niepowodzeniem",
   "backup.message_restore_help_one": "Przywrócenie backupu odtworzy całą konfiguracje utworzoną za pomocą Homebridge UI wraz z accessorimai, plikiem konfiguracyjnym i wszystkimi wtyczkami zainstalowanymi z repozytorium npm.",
   "backup.message_restore_help_two": "Po przywróceniu backupu trzeba ponownie zalogować się do Homebridge UI z danymi użytkownika zawartymi w tej kopii.",
-  "backup.message_restore_max_size": "Możesz przywrócić archiwum kopii zapasowej do rozmiaru: {{ maxBackupSizeText }}.",
+  "backup.message_restore_max_size": "Możesz przywrócić backup do rozmiaru: {{ maxBackupSizeText }}.",
   "backup.message_restore_warning": "Przywrócenie backupu jest nieodwracalne i zastąpi całą obecną konfigurację Homebridge.",
   "backup.message_scheduled_backup_none_yet": "Nie utworzono jeszcze planowych backupów. Następny automatyczny backup zostanie wykonany o godzinie {{ backupTime }}.",
-  "backup.message_scheduled_backup_time": "Pełny backup konfiguracji Homebridge jest tworzony automatycznie każdego dnia o godzinie {{ backupTime }} i trzymany przez {{ dayCount }} dni.",
-  "backup.title_backup": "Backup - utwórz",
-  "backup.title_restore": "Backup - przywróć",
+  "backup.message_scheduled_backup_time": "Pełny backup konfiguracji Homebridge jest tworzona automatycznie każdego dnia o godzinie {{ backupTime }} i trzymany przez {{ dayCount }} dni.",
+  "backup.title_backup": "Backup",
```

Comment:
"Utwórz backup"

Reply:
But personaly i preffer "Backup - utwórz" and "Backup - przywróć"

Reply:
![IMG_6479](https://github.com/user-attachments/assets/32e29bef-9aa5-4b1a-8801-ed497cf2b09e)